src/reporters/lang/en.js

```diff
@@ -188,6 +188,8 @@ const messages = {
 
   execMissingCommand: 'Missing command name.',
 
+  dashDashDeprecation:
+    "Using -- to pass arguments to your scripts isn't required anymore. Doing this may cause issues in future versions.",
```

Comment:
It won't fail, it will just be forwarded to the binary as-is.

```
$> yarn test -- --no-coverage
```

will become

```
$> jest -- --no-coverage
```

instead of

```
$> jest --no-coverage
```

This is important because in some case you really want to prevent something from being interpreted as an option.